### PR TITLE
[MISUV-8258] Decommissioning Feature Switches

### PR DESCRIPTION
**Pull request by Mbrookesy**
Decommissioning the following feature switches:

- feature-switch.enable-forecast-calculation

- feature-switch.enable-cut-over-credit

- feature-switch.enable-what-you-owe-credit-amount

- feature-switch.enable-mfa-credits-and-debits

### CODE REVIEW

**Comment by mywyau** [comment]
not sure if it's been done think it we should clean these up from the application.conf and diff env app configs - qa, satging etc.

File: it/test/controllers/PaymentHistoryControllerISpec.scala [hunk 1]
```diff
@@ -105,13 +103,10 @@ class PaymentHistoryControllerISpec extends ComponentSpecBase {
         elementTextBySelector("#refundstatus")(""),
       )
 
-      verifyAuditContainsDetail(PaymentHistoryResponseAuditModel(testUser, payments, cutOverCreditsEnabled = false,
-        mfaCreditsEnabled = false).detail)
+      verifyAuditContainsDetail(PaymentHistoryResponseAuditModel(testUser, payments).detail)
     }
 
     "return payment from earlier tax year description when CutOverCreditsEnabled and credit is defined" in {
```
Comment (AdamSmithHMRC) [inline it/test/controllers/PaymentHistoryControllerISpec.scala:109]: Can `CutOverCreditsEnabled` be removed from here?

**Comment by Mbrookesy** [comment]
Added the config changes as well

https://github.com/hmrc/app-config-base/pull/11142
https://github.com/hmrc/app-config-staging/pull/16130
https://github.com/hmrc/app-config-qa/pull/23633
https://github.com/hmrc/app-config-production/pull/23608

**Review by AdamSmithHMRC — APPROVED** [review]
Approved

**Pull request merged**